examples/handling/interface/topoints.py

```diff
@@ -0,0 +1,27 @@
+"""
+Raster to points
+================
+
+This example demonstrates the conversion of a raster to point vector using :func:`geoutils.Raster.to_points`.
+"""
+# sphinx_gallery_thumbnail_number = 2
```

Comment:
This line gets rendered, it probably shouldn't be?

Reply:
Solved in [9d487d0](https://github.com/GlacioHack/geoutils/pull/351/commits/9d487d0a85fe2acffbeb4b75d2e3765d23090f66)